src/containers/SearchPage/components/form/CheckboxSelector.tsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2022-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+import { ToggleItem } from '@ndla/ui/lib/Filter';
```

Comment:
Er denne kun tilgjengelig under lib/Filter? Hadde det ikkje vore meir logisk å importere fra `@ndla/ui`?

Reply:
Det er en blanding av intellij sin auto-import og at jeg ikke så at den ikke var eksponert i frontend-packages.
Lager en PR i frontend-packages for å eksponere den ut av @ndla-ui 👍